content/en/docs/instrumentation/java/_index.md

```diff
@@ -71,10 +71,6 @@ dependencies {
 }
 ```
 
-### API reference
-
-- [Javadoc](https://www.javadoc.io/doc/io.opentelemetry)
-
```

Comment:
Removing this since the API reference now appears in the auto-generation section contents at the end of the page. See the second screenshot in the opening comment for an illustration of this.